hubble/executor/hubapi.py

```diff
@@ -250,3 +251,17 @@ def exist_local(uuid: str, tag: str = None) -> bool:
         return True
     except FileNotFoundError:
         return False
+
+
+def load_config(yaml_path: Path) -> Dict:
+    """Load config of executor from YAML file."""
+    with open(yaml_path / 'config.yml') as fp:
+        tmp = yaml.safe_load(fp)
+
+    return tmp
+
+
+def get_tag_from_dist_info_path(path: Path) -> str:
+    """Get tag from stem of path."""
+    stem = path.stem
+    return stem.replace(r'.dist-info', '') if stem else None
```

Comment:
Should these two functions live in `helper.py` instead?

Reply:
> Should these two functions live in `helper.py` instead?

mmm, I think the the function `load_config`  should in `hubapi.py`, because it looks like `load_secret` loads the folder `~/.cache/jina/hub-package` info.

and maybe the function `get_tag_from_dist_info_path` should in `helper.py`, it just handle strings.